packages/template-mrt-reference-app/app/ssr.js

```diff
@@ -164,6 +173,15 @@ const tlsVersionTest = async (_, res) => {
     res.send(JSON.stringify({'tls1.1': response11, 'tls1.2': response12}, null, 4))
 }
 
+/**
+ * Express handler that returns all the environment variables set
+ */
+const envVarsEndpoint = async (_, res) => {
+    res.header('Content-Type', 'application/json')
+    const exposed = filterAndSortObjectKeys(process.env, ENVS_TO_EXPOSE)
+    res.send(JSON.stringify(exposed, null, 4))
```

Comment:
Non-blocking: did we consider using `res.json(...)` here?

We're already setting formatting for JSON responses from this app: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/6ab180b05774766cd40a3e0b2a5b17891dcbc3b2/packages/template-mrt-reference-app/app/ssr.js#L258

Reply:
Done. The wildcard approach makes sense. I've also removed the /env-vars endpoint since / already returns the env vars. Addressed in this commit: [d16593e](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1207/commits/d16593eda3a2c3ec80bb9a3f5e4f7c16e558664c)

Deployed and tested the app to: https://p5t-e9t.mobify-storefront-staging.com/